hrm-service/src/permissions/helpers.ts

```diff
@@ -0,0 +1,8 @@
+import { User } from './index';
```

Comment:
Could we try to make all the TS conversions git moves rather than delete & creates? For one thing it would make them a lot easier for me to review 😛

Reply:
Yes I've used `git mv` for all of them, but sometimes this happens where git shows the diff as deletion addition (I know this also happens to @murilovmachado) :( I don't really know why or how can I prevent that from happening but happy to do so if you have any idea.